script/copy_deduplicate

```diff
@@ -12,12 +12,13 @@ or to process only a specific list of tables.
 
 from argparse import ArgumentParser
 from datetime import datetime, timedelta
-from fnmatch import fnmatch
 from itertools import groupby
+from google.cloud import bigquery
 from multiprocessing.pool import ThreadPool
 from uuid import uuid4
 
-from google.cloud import bigquery
+import fnmatch
+import re
```

Comment:
I'm not sure why these were moved around. This was previously following the convention of having stdlib imports together alphabetically, blank line, then external imports.

Reply:
i wasn't sure which way to order them, so i picked one. i'll switch it.